spot_driver/include/spot_driver/api/default_spot_api.hpp

```diff
@@ -19,7 +20,7 @@ namespace spot_ros2 {
 
 class DefaultSpotApi : public SpotApi {
  public:
-  explicit DefaultSpotApi(const std::string& sdk_client_name,
+  explicit DefaultSpotApi(const std::string& sdk_client_name, const std::chrono::seconds& timesync_timeout,
```

Comment:
```suggestion
  explicit DefaultSpotApi(const std::string& sdk_client_name, const std::chrono::seconds timesync_timeout,
```

nit: there's some inconsistency in the PR with the input parameter being a reference or not. I think generally for `chrono` duration types, there isn't a performance gain from taking a reference since the memory footprint of a duration object is the same size as a reference.

Reply:
oops, yeah, doesn't matter in this case. thanks, changed it.